dev/SplitButton/SplitButton.xaml

```diff
@@ -281,9 +281,10 @@
                             <Button.Content>
                                 <TextBlock
                                     FontFamily="Segoe MDL2 Assets"
```

Comment:
Has any discussion happened about if this is now the time to start replacing Segoe MDL2 Assets with FluentIcons?

Reply:
@chigy

Reply:
@mdtauk , we are looking to update the icon assets.

Reply:
Microsoft Office only recently made new icons that are based on the Segoe MDL2 symbols. It still looks very WPF-ey too. 

The new pressed state for ToggleSwitch is totally not inspired by iOS >.>